jgiven-core/src/main/java/com/tngtech/jgiven/impl/ScenarioModelBuilder.java

```diff
@@ -159,6 +171,18 @@ public void introWordAdded( String value ) {
         introWord.setValue( value );
     }
 
+    private void fillerWordAdded( String value ) {
+        fillerWords.add( new Word( value ) );
+    }
+
+    private void punctuationAdded( String value ) {
+        if ( fillerWords.isEmpty() ) {
```

Comment:
I am not entirely certain about the point of this clause here. If I understood it correctly, then `fillerWord` is prepended before the step method words. I.e. if fillerwords were empty but I had punctuation then I would have a dot at the beginning of the sentence. Moreover, the punctuation would be treated as its own word, contrary to how it is handled when there are words present.

Reply:
--obsolete due to removal of the `@Punctuation` annotation